l10n_ch_base_bank/tests/test_bank.py

```diff
@@ -158,6 +158,7 @@ def test_postal_without_bank(self):
 
     def test_iban_postal(self):
         bank_acc = self.new_form()
+        bank_acc.bank_id = self.post_bank
```

Comment:
This is not the mistake, you can revert this commit. The error comes at line 31 where it searchs for the postal account with BIC POFICHBEXXX. It should find one unique record but for a reason, it finds 2. It's maybe because you have duplicate data between your .yml demo file and .xml demo file. If you remove the .yml it will maybe fix it or try to find where in another place there would already be a demo record for the Postfinance bank account.